go/vt/vttablet/tabletmanager/vreplication/controller.go

```diff
@@ -268,11 +268,11 @@ func (ct *controller) runBlp(ctx context.Context) (err error) {
 			isUnrecoverableError(err) ||
 			!ct.lastWorkflowError.ShouldRetry() {
 
-			log.Errorf("vreplication stream %d going into error state due to %+v", ct.id, err)
 			if errSetState := vr.setState(binlogdatapb.VReplicationWorkflowState_Error, err.Error()); errSetState != nil {
-				log.Errorf("INTERNAL: unable to setState() in controller. Attempting to set error text: [%v]; setState() error is: %v", err, errSetState)
+				log.Errorf("INTERNAL: unable to setState() in controller: %v, Could not set error text to: %v.", errSetState, err)
```

Comment:
Nit, but the `Could` in the middle of the log message should be lowercase `could` or we should end the previous sentence: `... controller: %v.` It's obviously minor / nit picky and either one is fine.